plugins/backend/ttrss/ttrssAPI.vala

```diff
@@ -20,11 +20,26 @@ public class FeedReader.ttrssAPI : GLib.Object {
 	private string m_ttrss_sessionid;
 	private uint64 m_ttrss_apilevel;
 	private Json.Parser m_parser;
+	private string? m_iconDir = null;
```

Comment:
Why does this backend use a different icon directory than the one all of the others use?